spec/footer_spec.rb

```diff
@@ -8,6 +9,7 @@
     @header.click_my_account
     @logon.login('akratz@mcfadyen.com', 'Admin123') # info provisória
```

Comment:
The login will not work with the current login on site now. You will need to automate login on site based on the login function.